lib/apiv2/doc.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+Package api implements the struct definitions used on the northbound client API.
+
+In particular this includes:
+	-  The various resource types as exposed on the Northbound API.  The valid resource
+	   types are:
+		-  BGPPeer
+		-  HostEndpoint
+		-  Policy
+		-  Pool
+		-  Profile
```

Comment:
Isn't this list missing some types? WEP, etc?

Reply:
yup and needs GNP and NP